docs/cli/data_format.rst

```diff
@@ -50,7 +10,7 @@ both :ref:`libmultilabel-format` and
 .. _libmultilabel-format:
 
 LibMultiLabel Format
-^^^^^^^^^^^^^^^^^^^^
+--------------------
 
 The LibMultiLabel format is a format for labels,
 raw texts and optional IDs. It combines them
```

Comment:
They are combined in a single file with tables ... It must satisfies the following requirements
- one ...
- ID, ... are separated by ...
- labels ...
- each field should not ... <TAB>

An example with the ...

...

An example without the ...